NDB.Covid19/NDB.Covid19/Configuration/OAuthConf.cs

```diff
@@ -2,11 +2,12 @@ namespace NDB.Covid19.Configuration
 {
     public static class OAuthConf
     {
-        public static string OAUTH2_CLIENT_ID = "INJECTED_IN_APP_CENTER_DURING_BUILD";
+        public static string OAUTH2_CLIENT_ID = "smittestopp";
         public static string OAUTH2_SCOPE = "openid smittestop";
-        public static string OAUTH2_REDIRECT_URL = "INJECTED_IN_APP_CENTER_DURING_BUILD";
-        public static string OAUTH2_AUTHORISE_URL = "INJECTED_IN_APP_CENTER_DURING_BUILD";
-        public static string OAUTH2_ACCESSTOKEN_URL = "INJECTED_IN_APP_CENTER_DURING_BUILD";
+        public static string OAUTH2_REDIRECT_URL = "no.fhi.smittestopp-exposure-notification:/oauth2redirect";
```

Comment:
The same with these, but have this anything to do with local development (just guessing).  Are there differences in how you do local development that causes these to have got their own distinct settings?

Reply:
These are the appropriate values for development (except redirect uri which is the same in all environments). They should be replaced during build even if it doesn't state so